### DISCUSSION TITLE
file splitting for mp3

### DISCUSSION THREAD

**Discussion by timadye** [audiamus/AaxAudioConverter#203] [Ideas]
A nice extension to this program would accept MP3 files as input and just do the file splitting, without conversion.

This might seem a strange request, since it is outside the scope of the program (the hint is in the name! 😄), but this is the easiest and most reliable program I have found for splitting audio files for any format. Also, I got used to using it for Audible AAX files, so it's a pity I can't do the same also for audiobooks.com files, which are in Ogg Vorbis or MP3 format (user's choice). I'd like to encourage the use of a DRM-free format, but the AAX Audio Converter is too good, and makes splitting AAX files easier 😁. Can it do both?

Did I mention what a great program this is already?! 😎

(Sorry I posted first as #202, but I saw this is probably a more appropriate forum. I will close the issue.)

**Comment by audiamus** [comment]
I haven't tested this myself but you could try to rename that .mp3 file to .aa. .aa is the older Audible format which was some encrypted form of .mp3. FFmpeg, which does all the audio processing stuff for AAX Audio Converter may be able to read this file and handle it. And ATL.net, which processes the meta data, may be able to read embedded chapter data.

**Reply by timadye** [reply]
I only tried renaming to `.aax`. I'll try renaming to `.aa` instead and let you know if it works. It would be fantastic if it does!

**Reply by timadye** [reply]
Renaming `.mp3` to `.aa` works! The single file is split up just like converting an AAX file. Just what I wanted.

Thank you so much!

**Comment by timadye** [comment]
To summarize, a nice workaround is to rename the `.mp3` file to `.aa`, and then open/drop the file in AaxAudioConverter.

It would be nice if AaxAudioConverter could accept `.mp4` (#201) and `.mp3` directly, instead of having to rename to `.aax` and `.aa` respectively. Maybe in a future version? In the meantime, renaming the files is a perfectly good workaround.
